srcpkgs/nextcloud-client/template

```diff
@@ -0,0 +1,54 @@
+# Template file for 'nextcloud-client'
+pkgname=nextcloud-client
+version=2.6.0
+revision=1
+wrksrc="desktop-${version}"
+build_style=cmake
+configure_args="-Wno-dev"
+hostmakedepends="pkg-config"
+makedepends="qt5-tools-devel qt5-webengine-devel qt5-declarative-devel
+ qt5-webchannel-devel qt5-location-devel qtkeychain-qt5-devel sqlite-devel
+ qt5-webkit-devel libcloudproviders-devel $(vopt_if dolphin 'extra-cmake-modules kio-devel')"
+conf_files="/etc/Nextcloud/sync-exclude.lst"
+short_desc="NextCloud Desktop client"
+maintainer="yopito <pierre.bourgin@free.fr>"
+license="GPL-2.0-or-later"
+homepage="https://nextcloud.com/clients/"
+distfiles="https://github.com/nextcloud/desktop/archive/v${version}.tar.gz"
+checksum=7b3f3c14d2e44826a5183fd59a7412c6dd5ed00296873e35c566f75c14c3a3ea
+
+build_options="dolphin"
+desc_option_dolphin="Build KDE dolphin support"
+build_options_default="dolphin"
+
+case "$XBPS_TARGET_MACHINE" in
+	armv6*|armv7*) broken="qt5-tools-devel unavailable" ;;
+esac
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-qmake qt5-host-tools qt5-tools"
+	# provides desktoptojson
+	hostmakedepends+=" $(vopt_if dolphin 'kcoreaddons')"
+fi
+
+if [ $build_option_dolphin ]; then
```

Comment:
Using `subpackages` for this is probably better than putting an if statement around a function definition.